gotestcover_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
Can you remove this import/dependency and add simple tests?
This code is so trivial it shouldn't require a third party library.

``` go
if err != nil {
	t.Fatal(err)
}
if flagCPU != "4" {
	t.Errorf("flagCPU is not equal to 4, got %s", flagCPU)
}
```